### PR TITLE
fix(storefront): BCTHEME-457 Update focus tooltip styles contrast to achieve accessibility AA Complaince

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
@bc-alexsaiannyi @BC-tymurbiedukhin @yurytut1993 @bc-azvierieva 
#### What?

This PR fixes the problem of low contrast of the tooltip to achieve accessibility AA Complaince. Updated color for tooltip in dark CornerStone theme.

#### Tickets / Documentation

- [BCTHEME-457](https://jira.bigcommerce.com/browse/BCTHEME-457)

#### Screenshots (if appropriate)
Bold(dark) theme:
tooltip-bold-theme

contrast-checker-bold-theme

Light and Warm themes:
tooltip-light-theme
tooltip-warm-theme

contrast-checker-light-and-warm-theme

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -309,6 +309,8 @@
     "color_badge_product_sold_out_badges": "#007dc6",
     "color_text_product_sold_out_badges": "#ffffff",
     "color_hover_product_sold_out_badges": "#000000",
+    "tooltip-textColor--focus": "#ffffff",
+    "tooltip-backgroundColor--focus": "#313440",
```
Comment (BC-tymurbiedukhin) [inline config.json:313]: config.json file includes page builder setting, so we need to define the approach we use here:
- using internal scss var 
- implement full cycle of customising with page builder

Comment (BC-tymurbiedukhin) [inline config.json:313]: Agree with Yury, modificator should be removed from tooltip name

**Review by BC-tymurbiedukhin — COMMENTED** [review]
Could you please add changelog item as well (see CHANGELOG.md for ref)?

File: assets/scss/common/_focus-tooltip.scss [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
+            border-color: transparent transparent $tooltip-backgroundColor-focus transparent;
         }
 
         &:after {
             content: attr($attr);
             padding: remCalc(4px) remCalc(6px);
-            background-color: $adminBar-tooltip-bg-backgroundColor;
-            color: white;
+            background-color: $tooltip-backgroundColor-focus;
+            color: $tooltip-textColor-focus;
```
Comment (yurytut1993) [inline assets/scss/common/_focus-tooltip.scss:16]: Var name here also looks like with "-focus" modifier. I believe we should better name it as focusTooltip-backgroundColor and focusTooltip-textColor, based on mixin name

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-alexsaiannyi — APPROVED** [review]

**Pull request merged**